renovate.json

```diff
@@ -4,6 +4,31 @@
   ],
   "ignorePaths": [".kokoro/requirements.txt"],
   "ignoreDeps": ["rules_pkg"],
+  "regexManagers": [
+    {
+      "fileMatch": ["^gax-java/dependencies\\.properties$"],
+      "matchStrings": ["=(?<depName>.+\\:.+?):(?<currentValue>.+?)\\n"],
```

Comment:
As per the documentation (https://docs.renovatebot.com/modules/manager/regex/#regular-expression-capture-groups) I confirmed the regex "=(?<depName>.+\\:.+?):(?<currentValue>.+?)\\n" works in https://regex101.com/r/01eHFC/1